modules/react/common/spec/createCanvasTheme.spec.tsx

```diff
@@ -51,6 +61,15 @@ describe('createCanvasTheme', () => {
       contrast: '#494949',
     };
 
+    expected.palette.action = {
+      lightest: '#ffff7d',
+      light: '#ffd64a',
+      main: 'orange',
```

Comment:
Is this supposed to be a hex value?

Reply:
it's just a test, so orange is still a valid css color.